Netable/Example/ExampleApp.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ExampleApp.swift
+//  Example
+//
+//  Created by Amy Oulton on 2022-11-23.
+//  Copyright © 2022 Steamclock Software. All rights reserved.
+//
+
+import SwiftUI
+
+@main
+struct ExampleApp: App {
+    init() {
+        ExampleNetworkService.shared
```

Comment:
This is likely *not* an ideal way to render this network out, but is serving as a simple way to do it for right now while building. This can likely be improved upon before completing the project.